pom.xml

```diff
@@ -38,14 +39,21 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-jooq</artifactId>
 		</dependency>
+		<!--
```

Comment:
please remove if unused.

Reply:
security dependencies should stay, the API should be reachable by basic auth token for now